### ISSUE TITLE
python build fails on linux/arm64

### ISSUE THREAD

**Issue by glehmann** [tree-sitter/workflows#2]
```
  Status: Downloaded newer image for quay.io/pypa/manylinux2014_aarch64:2024-01-23-12ffabc
  WARNING: The requested image's platform (linux/arm64/v8) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested
  c9eb7bf0f6259a7e286b7c0420cf949a5c0d8b6fe3f37aa1f8886bc3899d2b58
      + /bin/true
  exec /usr/local/bin/manylinux-entrypoint: exec format error
  Error: The operation was canceled.
```

maybe there should be a qemu install step?

```
      - name: Set up QEMU
        id: qemu
        uses: docker/setup-qemu-action@v1
        with:
          image: tonistiigi/binfmt:latest
          platforms: all

```

see https://github.com/glehmann/tree-sitter-earthfile/actions/runs/8523501164/job/23345908752#step:4:488 for an example.

**Comment by ObserverOfTime** [comment]
Fixed in 40bd608aa33e574baf89ed4290db19751df3bccf. It should work if you rerun the failed job.

**Comment by glehmann** [comment]
It works for me, thanks!
